### PR TITLE
Adding README and License

### PR DESCRIPTION
**Pull request by TornaxO7**
Closes https://github.com/eneoli/flakeshot/issues/15

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2012-2023 Scott Chacon and others
```
Comment (eneoli) [inline LICENSE.txt:1]: That's neither my nor your name

Comment (eneoli) [reply LICENSE.txt:1]: Also add `MIT License` at the top please.

**Review by eneoli — CHANGES_REQUESTED** [review]

File: LICENSE.txt (hunk 1 above)
Comment (TornaxO7) [reply LICENSE.txt]: aye! Fixed

File: README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# What is `flakeshot`?
+
+`flakeshot` is a screenshot-tool for unix-systems which runs natively on `wayland` *and* `x11`.
+
+# Motivation
+We all know some screenshot tools like [flameshot] (for X11) or [grim] (for wayland) but they they
```
Comment (eneoli) [inline README.md:6]: they they at the end

**Review by eneoli — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (TornaxO7) [reply README.md:6]: whoops

Comment (TornaxO7) [reply README.md]: fixed

File: README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# What is `flakeshot`?
+
+`flakeshot` is a screenshot-tool for unix-systems which runs natively on `wayland` *and* `x11`.
```
Comment (eneoli) [inline README.md:3]: Wayland and X11 with capital letters

File: README.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+# Motivation
+We all know some screenshot tools like [flameshot] (for X11) or [grim] (for wayland) but they
```
Comment (eneoli) [inline README.md:6]: Wayland capital letter

File: README.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+were often only either for X11 *or* wayland. `flakeshot` should close this gap by being a screenshot
+tool which should run on X11 as good as on wayland.
+
+# Status
+`flakeshot` is still under heavy development and not considered useable.
+
+# License
+See [LICENSE.txt].
+
+[flameshot]: https://github.com/flameshot-org/flameshot
```
Comment (eneoli) [inline README.md:16]: Add a header like "Similar projects"

Comment (eneoli) [reply README.md:16]: Also why is there a LICENSE.txt under grim?

File: README.md (hunk 3 above)
Comment (eneoli) [inline README.md:3]: Either with - or without

File: README.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
+# What is `flakeshot`?
```
Comment (eneoli) [inline README.md:1]: Maybe add a header "flakeshot"

**Review by eneoli — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (TornaxO7) [reply README.md:3]: fixed

File: README.md (hunk 4 above)
Comment (TornaxO7) [reply README.md:6]: fixed

File: README.md (hunk 6 above)
Comment (TornaxO7) [reply README.md:1]: I removed the header

File: README.md (hunk 3 above)
Comment (TornaxO7) [reply README.md]: fixed (choosed without)

File: README.md (hunk 5 above)
Comment (TornaxO7) [reply README.md:11]: fixed

**Review by eneoli — APPROVED** [review]

**Pull request merged**